### PR TITLE
product: mezzanine: shiratech cam 1

### PR DESCRIPTION
**Pull request by ric96**
Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

**Comment by ric96** [comment]
Broken links may be fixed by merging the doc pr: https://github.com/96boards/documentation/pull/915

**Review by sgermanserrano — APPROVED** [review]
LGTM

File: _product/mezzanine/shiratech-cam1/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+title: Shiratech Camera 1 Mezzanine
+layout: product
+permalink: /product/shiratech-cam1/
+display_product: true
+shortname: shiratech-cam1
+archived: false
+dateAdded: 2020-11-23 09:00:00+00:00
+description: |-
+    The new Shiratech Camera 1 Mezzanine is a 96Boards compatible add on board with OnSemiconductor 13Mp imaging sensor and AP1302 ISP for evaluation and prototyping.
+tab_menu:
```
Comment (kylekirkby) [inline _product/mezzanine/shiratech-cam1/README.md:11]: This should be `sticky_tab_bar`

**Review by kylekirkby — CHANGES_REQUESTED** [review]

File: _product/mezzanine/shiratech-cam1/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+layout: product
+permalink: /product/shiratech-cam1/
+display_product: true
+shortname: shiratech-cam1
+archived: false
+dateAdded: 2020-11-23 09:00:00+00:00
+description: |-
+    The new Shiratech Camera 1 Mezzanine is a 96Boards compatible add on board with OnSemiconductor 13Mp imaging sensor and AP1302 ISP for evaluation and prototyping.
+sticky_tab_bar:
+  - tab_title: Shiratech Camera 1 Mezzanine
```
Comment (kylekirkby) [inline _product/mezzanine/shiratech-cam1/README.md:12]: `tab_title` needs to be `title` and `tab_link` should be `url`.

**Review by kylekirkby — CHANGES_REQUESTED** [review]

**Review by kylekirkby — APPROVED** [review]

**Pull request merged**